### ISSUE TITLE
Figure out a way to show CBOR objects embedded as byte strings in extended diagnostic notation

### ISSUE THREAD

**Issue by cabo** [cbor-wg/cddl#2]
(As long as an extended diagnostic notation is being developed as part of the CDDL spec:)

It would be nice to have a notation for byte strings that are really encoded CBOR items or sequences of such (matching what .cbor and .cborseq do at the CDDL level).

One motivating example:
https://github.com/core-wg/oscoap/pull/100

**Comment by cabo** [comment]
Strawman:

turn
```
h'a2044125064105'   / { 4:h'25', 6:h'05' } /
```

into

```
<< { 4:h'25', 6:h'05' } >>
```

To discuss: choice of delimiters, .cbor vs. .cborseq (just add commas for the latter?).

**Comment by cabo** [comment]
Was already addressed in afd9abe5; syntax was tried out (and seems to work well enough) in draft-ietf-ace-cbor-web-token
